content/1_General/IO.mdx

```diff
@@ -277,43 +280,10 @@ public class Main {
  * @author: Kattis
  */
 
-import java.io.*;
-import java.util.StringTokenizer;
-
-class Kattio extends PrintWriter {
-	private BufferedReader r;
-	private StringTokenizer st;
-
-	// standard input
-	public Kattio() { this(System.in,System.out); }
-	public Kattio(InputStream i, OutputStream o) {
-		super(o);
-		r = new BufferedReader(new InputStreamReader(i));
-	}
-	// USACO-style file input
-	public Kattio(String problemName) throws IOException {
-		super(new FileWriter(problemName+".out"));
-		r = new BufferedReader(new FileReader(problemName+".in"));
-	}
-
-	// returns null if no more input
-	public String next() {
-		try {
-			while (st == null || !st.hasMoreTokens())
-				st = new StringTokenizer(r.readLine());
-			return st.nextToken();
-		} catch (Exception e) {}
-		return null;
-	}
-
-	public int nextInt() { return Integer.parseInt(next()); }
-	public double nextDouble() { return Double.parseDouble(next()); }
-	public long nextLong() { return Long.parseLong(next()); }
-}
-
 public class Main {
-	static Kattio io = new Kattio();
+	CodeSnip{Kattio}
 	public static void main(String[] args) {
+		Kattio io = new Kattio();
```

Comment:
since this is the first time Kattio appears and the entire code block is for kattio, I don't think we should snippet this?